service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/AddressMapper.java

```diff
@@ -0,0 +1,48 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import java.util.Arrays;
+import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+import uk.nhs.connect.iucds.cda.ucr.AD;
+
+import org.hl7.fhir.dstu3.model.Address;
+
+public class AddressMapper {
+
+    public static Address mapAddress(AD itkAddress) {
```

Comment:
Please don't make these methods static. Let's use Spring Components. This way we can easily test each of them separately.

Reply:
Good idea :)